app/src/main/java/com/eveningoutpost/dexdrip/g5model/Ob1DexTransmitterBattery.java

```diff
@@ -93,6 +95,9 @@ public boolean voltageAWarning() {
     }
 
     public boolean voltageBWarning() {
+        if (shortTxId()) { // G7 only
+            return voltageB() < (G5BaseService.LOW_BATTERY_WARNING_LEVEL - 25);
```

Comment:
it would be best if 25 is a constant

Reply:
This is a copy of a code that uses 10, which was not a constant.  I just followed that practice.